_gsocproposals/2019/proposal_AllenSIMTtoSPMD.md

```diff
@@ -0,0 +1,69 @@
+
+---
+project: Allen
+title: SIMT to SPMD code translation
+layout: gsoc_proposal
+year: 2019
+organization: CERN
+---
+
+## Description
```

Comment:
Just some typos:

- 'The entirety of algorithms in Allen are written' => 'The entirety of algorithms in Allen is written'
- 'The SIMT programming model present' => 'The SIMT programming model presents'

Reply:
The second one is not a typo, "present" is an adjective here. I will change it so it doesn't lead to errors.

Reply:
@dcampora: I read too fast … sorry!